README.md

```diff
@@ -87,6 +88,34 @@ For further information, please visit the [Airflow Wiki](https://cwiki.apache.or
 
 Official container (Docker) images for Apache Airflow are described in [IMAGES.rst](IMAGES.rst).
 
+## Installing from PyPI
```

Comment:
Do you think it's worth adding information about installing the master version (from github)?